index.html

```diff
@@ -487,6 +494,10 @@ <h3>Current</h3>
                   <img src="https://github.com/lilymcmullen.png" style="max-height: 188px;" class="img-fluid img-thumbnail" alt="Lily McMullen headshot">
                   <p class="mt-2 align-text-bottom"><span class="lead"><a href="https://github.com/lilymcmullen" class="stretched-link" target="_blank" rel="noopener noreferrer">Lily McMullen</a></span></p>
                 </div>
+                <div class="col-6 col-md-4 col-lg-3">
+                  <img src="https://github.com/joeybish.png" style="max-height: 188px;" class="img-fluid img-thumbnail" alt="Beautiful Joey headshot">
```

Comment:
```suggestion
                  <img src="https://github.com/joeybish.png" style="max-height: 188px;" class="img-fluid img-thumbnail" alt="Joey Bishop headshot">
```
Since this is alt text we'll want to keep it simple and consistent. It is your headshot though, so totally up to you. 😄